distro/run/pom.xml

```diff
@@ -18,13 +18,11 @@
   <dependencyManagement>
     <dependencies>
 
-      <!-- https://jira.camunda.com/browse/CAM-13717 -->
+      <!-- Explicit override tomcat version before spring-boot-dependencies -->
```

Comment:
🙃 How does this comment relate to snakeyaml?

Reply:
It should say Snakeyaml, it's a copy-paste mistake.